eodag/rest/utils.py

```diff
@@ -731,7 +732,14 @@ def get_stac_catalogs(url, root="/", catalogs=[], provider=None, fetch_providers
     ).get_stac_catalog()
 
 
-def search_stac_items(url, arguments, root="/", catalogs=[], provider=None):
+def search_stac_items(
```

Comment:
please rebase to get type hints from `develop` branch

Reply:
done